src/AppBundle/Controller/AdmissionAdminController.php

```diff
@@ -268,4 +270,20 @@ public function showApplicationAction(Application $application)
             'application' => $application,
         ));
     }
+
+    public function showTeamInterestAction(Semester $semester)
+    {
+        $user = $this->getUser();
+        $department = $semester->getDepartment();
+
+        if ($user->getDepartment() !== $department) {
```

Comment:
Nå får teamledere 403: Access Denied når de prøver å se teaminteresse til en annen avdeling. Hvis dette er riktig bør ikke knappen for teaminteresse vises når man ser på en annen avdeling. 

Hvis det er meningen at teamledere skal kunne se andre avdelinger sin teaminteresse kan dette kanskje fungere: `!$this->isGranted(Roles::TEAM_LEADER) && $user->getDepartment() !== $department`